livekit-agents/livekit/agents/llm/fallback_adapter.py

```diff
@@ -73,6 +73,17 @@ def __init__(
             _LLMStatus(available=True, recovering_task=None) for _ in self._llm_instances
         ]
 
+        for llm_instance in self._llm_instances:
+            llm_instance.on("metrics_collected", self._on_metrics_collected)
+
+    @property
+    def model(self) -> str:
+        return "FallbackAdapter"
```

Comment:
So, when using FallbackAdapter, is it impossible to obtain the actual model being used?